src/jit/block.h

```diff
@@ -438,6 +438,8 @@ struct BasicBlock : private LIR::Range
 #define BBF_CLONED_FINALLY_BEGIN 0x100000000 // First block of a cloned finally region
 #define BBF_CLONED_FINALLY_END 0x200000000   // Last block of a cloned finally region
 
+#define BBF_DOMINATED_BY_NOT_NORMAL_ENTRY 0x400000000 // Full set of dominators contains not normal entry.
+
```

Comment:
It means that we want to SetDominatedByExceptionalEntryFlag for fgEnterBlks (it includes normal entry and blocks from EH table), but also for finally and special code EH.
Does it sound right?